### PR TITLE
[TS] LPS-183708 - we should default to language.properties if a locale does not have a specific corresponding translation

### PR DESCRIPTION
**Pull request by wanderlast**
This fix addresses an issue where we currently default to the default locale language bundle instead of the Language.properties file (which is what we should be defaulting to, per our documentation). The fix involves making sure that we include the default Language.properties file when loading our language resources. A test is included as well. For more context see the PTR here: https://liferay.atlassian.net/browse/PTR-3987

Please let me know if there's any questions or concerns. Thanks!

CC: @ericyanLr

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:friendly-url

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:stop

**Comment by Kvale1733** [comment]
ci:test:friendly-url

**Comment by zhangyan0701** [comment]
No regressions found. And the manual testing for https://liferay.atlassian.net/browse/LPS-183708 passed.

**Comment by tinatian** [comment]
@wanderlast I am confused about your test commits https://github.com/liferay-core-infra/liferay-portal/pull/5222/commits/9d6f525a6e34d9abf0fb2490268c932f3a2ae3a5 and https://github.com/liferay-core-infra/liferay-portal/pull/5222/commits/00ebc3eb5ade8c360a2f754f481de6e3f1c30a6c.

In the first commit, you add a test.
In the second commit, you modify the one you added, and add another one. 

Please do not patch your own code in another commit, we have this rule in our pull request guideline at https://liferay.atlassian.net/wiki/spaces/ENGCOREINFRA/pages/2051376221/Pull+Requests+-+Guidelines. 

> If there is something wrong with one commit don’t create a new commit to patch it, fix it directly.

Please simplify your test and resend.

Thanks.
Tina.

File: modules/apps/portal-language/portal-language-test/src/testIntegration/java/com/liferay/portal/language/test/LanguageResourcesTest.java [hunk 1]
```diff
@@ -120,19 +134,64 @@ public void testLanguageResourceServiceTrackerCustomizerServiceRanking() {
...
+		Locale locale = new Locale("ps", "AF");
+
+		Assert.assertEquals(
+			expectedBaseLanguagePropertiesTranslation,
+			_language.get(locale, key, null));
+
+		_serviceRegistration1 = _register(
+			_VALUE_1, 0, _language.getLanguageId(locale));
+
+		_assertValue(_VALUE_1, locale);
```
Comment (tinatian) [inline modules/apps/portal-language/portal-language-test/src/testIntegration/java/com/liferay/portal/language/test/LanguageResourcesTest.java:169]: @wanderlast This part was used to test service tracker, why you need to test it in your context?

**Comment by wanderlast** [comment]
Hi @tinatian,
I did it in this manner to give credit to the work Eric did on the refactors, but I will resend if that's what's preferred. Thanks!

**Pull request closed without merging**